### ISSUE TITLE
sp.replit.com seems to be required for captchas

### ISSUE THREAD

**Issue by salad612** [hagezi/dns-blocklists#4054]
### Which AdBlocker/DNS cloud service do you use?

NextDNS

### ControlD users

- [ ] IMPORTANT - I assure that I have not set the Block Response in ControlD to Custom or Branded and can confirm that the problem still occurs.

### NextDNS users

- [X] IMPORTANT - I can assure that I disabled the block page in NextDNS and can confirm that the problem still occurs.

### With which block list(s) does the problem occur?

Multi PRO++

### Which domain(s) should be unblocked?

sp.replit.com

### Why should the domain(s) be unblocked?

while attempting to sign in (or even submitting a forgot password request) without adding this domain to the allowlist I was getting the error message "Your captcha token is invalid. Please refresh the page and try again." even though no captchas showed up. looked in the logs, found this domain was getting blocked, and added it to allowlist. now it doesnt show that error message. sometimes it says "captcha failed" without showing a captcha but if you press it again a few times it works. Other times it does show a captcha now.

I also saw this domain getting blocked: js.zi-scripts.com

but didnt try unblocking it as I no longer needed to.

### Privacy

- [X] I confirm that the report does not contain any private information.
